vendor/k8s.io/kube-openapi/pkg/handler3/handler.go

```diff
@@ -49,9 +52,20 @@ const (
 	subTypeJSON     = "json"
 )
 
+// OpenAPIV3Discovery is the format of the Discovery document for OpenAPI V3
+// It maps Discovery paths to their corresponding URLs with a hash parameter included
+type OpenAPIV3Discovery struct {
+	Paths map[string]OpenAPIV3DiscoveryGroupVersion
```

Comment:
if this is serializing to json is has to define json tags and should follow API conventions of being lower-cased (e.g. `paths` and `url`)

Reply:
@Jefftree Jordan's comments here in the `vendor` directory I think are more targeted towards your recent PR:

https://github.com/kubernetes/kube-openapi/pull/284